astropy/table/meta.py

```diff
@@ -165,7 +165,7 @@ def _get_col_attributes(col):
     attrs['name'] = col.info.name
 
     type_name = col.info.dtype.type.__name__
-    if six.PY3 and (type_name.startswith('bytes') or type_name.startswith('str')):
+    if not six.PY2 and (type_name.startswith('bytes') or type_name.startswith('str')):
```

Comment:
Probably out of scope for this PR but doesn't `startwith` allow using a `tuple` of `prefixes`? See also [`str.startswith`](https://docs.python.org/2/library/stdtypes.html#str.startswith)